dashboard/craco.config.js

```diff
@@ -11,11 +12,15 @@ module.exports = {
         // add the required polyfills (not included in webpack 5)
         new NodePolyfillPlugin({
           // Allow using console.log
-          excludeAliases: ['console'],
+          excludeAliases: ["console"],
         }),
         new webpack.ProvidePlugin({
-          process: 'process/browser.js',
-          Buffer: ['buffer', 'Buffer'],
+          process: "process/browser.js",
+          Buffer: ["buffer", "Buffer"],
+        }),
+        new MonacoWebpackPlugin({
+          // see https://github.com/microsoft/monaco-editor/tree/main/webpack-plugin
+          languages: ["yaml"],
```

Comment:
We are bundling monaco-editor with a single language, as it is the only one being used currently. It helps decrease the webpack build mem footprint.